neqo-http3/src/features/extended_connect/webtransport_session.rs

```diff
@@ -266,13 +268,15 @@ impl WebTransportSession {
                                 error: 0,
                                 message: String::new(),
                             },
+                            Some(headers),
                         );
                         SessionState::Done
                     } else {
                         self.events.session_start(
                             ExtendedConnectType::WebTransport,
                             self.session_id,
                             status,
+                            &headers,
```

Comment:
It looks like we don't need to clone here as this is a value that doesn't need to be used later.